frontend-react/src/App.tsx

```diff
@@ -2,7 +2,7 @@ import { toRelativeUrl } from "@okta/okta-auth-js";
 import { useIdleTimer } from "react-idle-timer";
 import { Suspense, useEffect, useRef } from "react";
 import { NetworkErrorBoundary } from "rest-hooks";
-import { useNavigate } from "react-router-dom";
+import { ScrollRestoration, useNavigate } from "react-router-dom";
```

Comment:
Is this supposed to restore a users scroll position upon page reload? I don't see it working when I reload a page with the example of: `http://localhost:3000/resources/api/documentation/data-model#order-and-result-data-elements`

Reply:
🤷‍♂️ Its react-router's official component that you're supposed to use, but like everything from that library its up in the air whether it'll work right.